lib/Models/Feature/Feature.ts

```diff
@@ -1,60 +1,38 @@
-import Entity from "terriajs-cesium/Source/DataSources/Entity";
-import ImageryLayer from "terriajs-cesium/Source/Scene/ImageryLayer";
 import { observable } from "mobx";
+import Entity from "terriajs-cesium/Source/DataSources/Entity";
 import Cesium3DTileFeature from "terriajs-cesium/Source/Scene/Cesium3DTileFeature";
 import Cesium3DTilePointFeature from "terriajs-cesium/Source/Scene/Cesium3DTilePointFeature";
+import ImageryLayer from "terriajs-cesium/Source/Scene/ImageryLayer";
+import { JsonObject } from "../../Core/Json";
+import { BaseModel } from "../Definition/Model";
+import { TerriaFeatureData } from "./FeatureData";
 
 const customProperties = ["entityCollection", "properties", "data"];
-/**
- * A feature is just a Cesium Entity, with observable (ie. knockout-tracked) properties added for
- * currentDescription and currentProperties. These are tracked so that the feature info updates as the clock time changes,
- * because the properties and description themselves do not change (they are functions of the time, whose values change).
- * Set these if needed from an event listener on the terria clock, eg.
- *       terria.clock.onTick.addEventListener(function(clock) {
- *           if (typeof feature.description.getValue === 'function') {
- *               feature.currentDescription = feature.description.getValue(clock.currentTime);
- *           };
- *           if (typeof feature.properties.getValue === 'function') {
- *               feature.currentProperties = feature.properties.getValue(clock.currentTime);
- *           };
- *       });
- *
- * @alias Feature
- * @constructor
- * @param {Object} [options] Object with the same properties as Cesium's Entity.
- * @extends Entity
- */
-export default class Feature extends Entity {
-  /**
-   * Gets or sets the current properties. This property is observable.
-   */
-  @observable currentProperties: any = undefined;
-
-  /**
-   * Gets or sets the current description. This property is observable.
-   */
-  @observable currentDescription: string | undefined = undefined;
 
-  /**
-   * Gets or sets counter objects used to trigger an update of the Feature Info Section,
-   * to allow custom components to self-update. The object keys are timeoutIds, and values are
-   * {reactComponent: ReactComponent, counter: Integer}.
-   * This property is observable.
-   */
-  @observable updateCounters: any = undefined;
+/** Terria wrapper around Cesium Entity
+ * Adds a few extra properties
+ * -
+ */
 
-  data: any;
+export default class Feature extends Entity {
```

Comment:
Can we change name to CesiumFeature or TerriaFeature and change to named export instead of default just to make it easier to reason from where it comes. We have turfjs and protomaps feature also

Reply:
Good idea - changed to `TerriaFeature`